### DISCUSSION TITLE
Updating the add-on

### DISCUSSION THREAD

**Discussion by Kumagawa97** [MikeMoolenaar/Migaku-Japanese-Addon-Updated#10] [Q&A] [answered]
Hi! If I want to update the add-on to the latest release what should I do? Is there a way to keep the configuration from the previous version (ver 1.2.3+1.0.1) or do I need to follow the same procedure used to install it the first time?
Thank you for your work btw, this add-on was (and still is) an incredible asset :)

**Comment by MikeMoolenaar** [comment, marked as answer]
Hi, I recommend to extract the new version from the ZIP file to the Anki addons folder and overwrite all files. Settings are saved in the `Anki2/addons21/Migaku-Japanese-Addon-Updated/meta.json`  file so you don't want to remove that one. This meta file is not present in the ZIP file so you won't have issues if you overwrite the files.

**Reply by MikeMoolenaar** [reply]
I'll also update the README soon with clear instructions, thanks for the question :)

**Reply by Kumagawa97** [reply]
Fantastic, thank you!
